Batch Issuance (#789)
italia/eidas-it-wallet-docs: docs/en/credential-issuance-endpoint.rst

      - **proof_type**: JSON string denoting the proof type. It MUST be `jwt`.
      - **jwt**: the JWT used as proof of possession.
    - [`OpenID4VCI`_].
  * - **proofs**

>     - REQUIRED when the ``proof`` parameter is absent. Object providing one or more proof of possessions of the cryptographic key material to which the issued Credential instances will be bound to. he proofs object MUST contain the following mandatory claims:

Reviewer: ```suggestion
    - REQUIRED when the ``proof`` parameter is absent. Object providing one or more proof of possessions of the cryptographic key material to which the issued Credential instances will be bound to. The proofs object MUST contain the following mandatory claims:
```

>       - **proof_type**: JSON string denoting the proof type. It MUST be `jwt`.

Reviewer 2: ```suggestion
      - **proof_type**: JSON string denoting the proof type. It MUST be set with`jwt`.
```

>       - **proof_type**: JSON string denoting the proof type. It MUST be set with`jwt`.

Reviewer 2: ```suggestion
      - **proof_type**: JSON string denoting the proof type. It MUST be set with `jwt`.
```

>       - **jwt**: an array of JWT, where each jwt is used as proof of possession.

Reviewer 2: ```suggestion
      - **jwt**: an array of JWT, where each element within the array is used as proof of possession.
```

    - **Description**
    - **Reference**
  * - **credentials**

>     - REQUIRED if ``lead_time`` and ``transaction_id`` are not present, otherwise it MUST NOT be present. It is an array of one or more issued Credentials. The elements of the array MUST be objects. The number of elements in the credentials array matches the number of keys that the Wallet Instance has provided either via the ``proof`` or ``proofs`` parameter of the Credential Request. It contains the following parameters:

Reviewer 2: ```suggestion
    - REQUIRED if ``lead_time`` and ``transaction_id`` are not present, otherwise it MUST NOT be present. It is an array of one or more issued Credentials. The number of elements in the Credentials array matches the number of keys that the Wallet Instance has provided either via the ``proof`` or ``proofs`` parameter of the Credential Request. The array MUST contain JSON objects, where each object MUST have exactly one member with the key ``credential``. The value of the ``credential`` member MUST be a string containing the encoded Credential, as further described below:
```

      - The Credential Issuer cannot fulfill the request because the requested Credential Format is not supported. Section 8.3.1 of [`OpenID4VCI`_].
    * - *400 Bad Request* [REQUIRED]
      - ``invalid_proof``
      - The Credential Issuer cannot fulfill the request because the ``proof`` or ``proofs`` parameter in the Credential Request is invalid or absent or or the key proof(s) does not contain the ``c_nonce`` value. Section 8.3.1 of [`OpenID4VCI`_].
    * - *400 Bad Request* [REQUIRED]
      - ``invalid_nonce``

>       - The Credential Issuer cannot fulfill the request because the ``proof`` parameter in the Credential Request uses an invalid nonce. Section 8.3.1 of [`OpenID4VCI`_].

Reviewer: ```suggestion
      - The Credential Issuer cannot fulfill the request because the ``proof`` or ``proofs`` parameter in the Credential Request uses an invalid nonce. Section 8.3.1 of [`OpenID4VCI`_].
```
